base/uk.ac.stfc.isis.ibex.synoptic/src/uk/ac/stfc/isis/ibex/synoptic/SynopticWriter.java

```diff
@@ -38,18 +39,21 @@
 public class SynopticWriter extends TransformingWriter<SynopticDescription, String> {
 	
 	private SettableUpdatedValue<Boolean> canSave = new SettableUpdatedValue<>();
+    private final Observable<String> schema;
 
-	public SynopticWriter(Writable<String> destination) {
+    public SynopticWriter(Writable<String> destination, Observable<String> schema) {
 		writeTo(destination);
 		canSave.setValue(destination.canWrite());
 		destination.subscribe(this);
+        this.schema = schema;
 	}
 	
+
 	@Override
 	protected String transform(SynopticDescription value) {
 		// Converts the raw synoptic description into XML
 		try {
-			return XMLUtil.toXml(value);
+            return XMLUtil.toXml(value, SynopticDescription.class, schema.getValue());
```

Comment:
If no schema is available. This will pass null. Implicitly, this means there is no validation

- If we don't want validation in this case, we should say so explicitly in the comments.
- I think in this case we either need to wait for the schema to become available, or log that it wasn't. In most cases I expect schema checking shouldn't be an issue but I worry about the case that the schema is never available because the observable isn't hooked up correctly and we'd never notice, making the schema check worthless.